src/views/Home.jsx

```diff
@@ -1,12 +1,58 @@
 import './Home.css';
+import { useCallback, useState } from 'react';
+import { checkListToken } from '../api/firebase';
+import { useNavigate } from 'react-router-dom';
+
+export function Home({ handleNewToken, setListToken, listToken }) {
+	const [token, setToken] = useState('');
+	const [listNotFound, setlistNotFound] = useState('');
+	const redirect = useNavigate();
+
+	if (listToken) {
+		redirect('/list');
+	}
```

Comment:
@tinazarb @louieknolle Future improvement we may want to consider... currently if I join an existing list then I can't switch to another existing list unless I clear out local storage manually.